src/codeflare_sdk/cluster/cluster.py

```diff
@@ -337,6 +337,67 @@ def get_current_namespace():  # pragma: no cover
         return "default"
 
 
+def get_cluster(cluster_name: str, namespace: str = "default"):
+    try:
+        config.load_kube_config()
+        api_instance = client.CustomObjectsApi()
+        rcs = api_instance.list_namespaced_custom_object(
```

Comment:
https://github.com/kubernetes-client/python/blob/045b189a282f5165426b841e9e3b16b2739d3837/kubernetes/client/api/custom_objects_api.py#L1580

Can you use this directly?

Reply:
I get `urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=80): Max retries exceeded with url: /apis/ray.io/v1alpha1/namespaces/default/rayclusters/quicktest` when calling the `get_namespaced_custom_object` so I think there is something wrong with this function